example/two-pods-two-distinct-nx-gzip/05-deployment-second.yaml

```diff
@@ -21,14 +22,10 @@ spec:
         args: [ "infinity" ]
         resources:
           limits:
-            memory: 100Mi
+          memory: 100Mi
```

Comment:
should be +2 spaces

`oc explain Pod.spec.containers.resources.limits` explains. an example https://github.com/IBM/power-device-plugin/blob/main/examples/03-deployment.yaml#L26